src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -81,117 +112,278 @@ public void focus() {
+        formatColumn.setCellValueFactory(cellData -> new javafx.beans.property.SimpleStringProperty(cellData.getValue()[1]));
+        exampleColumn.setCellValueFactory(cellData -> new javafx.beans.property.SimpleStringProperty(cellData.getValue()[2]));
+
+        commandSummaryData.clear();
+
+        commandSummaryData.addAll(
+                new String[]{"Add", "add n/NAME p/PHONE_NUMBER\n e/EMAIL a/ADDRESS b/BIRTHDAY [t/TAG]…​", "add n/James Ho p/22224444 \ne/jamesho@example.com a/123, \nClementi Rd, 1234665 b/11 09 2001 \nt/friend t/colleague"},
+                new String[]{"Clear", "clear", ""},
+                new String[]{"Delete", "delete INDEX", "delete 3"},
+                new String[]{"Edit", "edit INDEX [n/NAME] [p/PHONE_NUMBER]\n [e/EMAIL] [a/ADDRESS] [t/TAG]…​", "edit 2 n/James Lee\n e/jameslee@example.com"},
+                new String[]{"Find", "find KEYWORD [MORE_KEYWORDS]", "find James Jake"},
+                new String[]{"List", "list", ""},
+                new String[]{"Help", "help", ""},
+                new String[]{"Paid", "paid INDEX", "paid 3"},
+                new String[]{"Unpaid", "unpaid INDEX", "unpaid 3"}
+        );
+        commandSummaryData.removeIf(row -> row == null || row.length == 0 ||
+                (row[0].isEmpty() && row[1].isEmpty() && row[2].isEmpty()));
+
+        commandSummaryTable.setItems(commandSummaryData);
+
+        commandSummaryTable.setColumnResizePolicy(TableView.CONSTRAINED_RESIZE_POLICY);
+    }
 
-            Marks the specified person from the FART book as unpaid.
+    /**
+     * Updates the TextFlow with bold, large titles, regular body text, hyperlinks, and code formatting for text in backticks.
+     */
+    private void updateTextFlow(String content) {
+        helpContentFlow.getChildren().clear();
 
-            ### Exiting the program : `exit`
+        String[] parts = content.split("\n", 2);
+        String titleText = parts[0];
+        String bodyText = parts.length > 1 ? parts[1] : "";
 
-            Exits the program.
+        Text title = new Text(titleText + "\n");
+        title.setStyle("-fx-font-weight: bold; -fx-font-size: 18px;");
+        helpContentFlow.getChildren().add(title);
 
-            ### Saving the data
+        if (content.contains("Quick start")) {
+            Text beforeLink1 = new Text("\n1. Ensure you have Java ");
+            Hyperlink javaLink = new Hyperlink("https://www.oracle.com/java/technologies/downloads/#java17");
+            javaLink.setOnAction(e -> openHyperlink("https://www.oracle.com/java/technologies/downloads/#java17"));
 
-            FART data is saved in the hard disk automatically after any command that changes the data.
-             There is no need to save manually.
+            Text afterLink1 = new Text("\n\n2. Download the latest ");
+            Hyperlink jarLink = new Hyperlink("https://github.com/se-edu/addressbook-level3/releases");
```

Comment:
I believe the link should point to FART instead, not AB3 :/

Reply:
good catch, rectified accordingly thanks!